client/src/style.css

```diff
@@ -1107,35 +1094,32 @@ button {
   --tw-gradient-stops: var(--tw-gradient-from),var(--tw-gradient-to);
 }
 
-::-webkit-scrollbar-track {
-  background-color: transparent;
-  border-radius: 9999px;
+/* Webkit scrollbar */
+::-webkit-scrollbar {
+  height: 0.1em;
+  width: 0.5rem;
 }
 
 ::-webkit-scrollbar-thumb {
-  background-color: hsla(0,0%,0%,.1);
+  background-color: rgba(0, 0, 0, 0.1);
+  border-radius: 9999px;
 }
 
-::-webkit-scrollbar-thumb:hover {
-  background-color: hsla(0,0%,0%,.3);
+.dark ::-webkit-scrollbar-thumb {
+  background-color: hsla(0, 0%, 100%, 0.1);
 }
 
-.scrollbar-transparent::-webkit-scrollbar-thumb {
-  background-color: rgba(0, 0, 0, 0.1);
+::-webkit-scrollbar-track {
+  background-color: transparent;
+  border-radius: 9999px;
 }
 
-@media (prefers-color-scheme: dark) {
```

Comment:
Hey, was there a particular reason this was removed? This was the original fix for the issue described with lightmode, as it's dependent on the code preference of the window.

Reply:
Does `.dark` negate the need for this?

Reply:
Yes, your fix wasn’t working for me as expected so I changed the approach here